types/block.go

```diff
@@ -434,68 +434,129 @@ func (h *Header) StringIndented(indent string) string {
 
 //-------------------------------------
 
-// CommitSig is a vote included in a Commit.
-// For now, it is identical to a vote,
-// but in the future it will contain fewer fields
-// to eliminate the redundancy in commits.
-// See https://github.com/tendermint/tendermint/issues/1648.
-type CommitSig Vote
```

Comment:
@tessr the main change is here, right now on the hub the commitSig is the Vote which contains redundant info, so this PR is creating an actually `CommitSig` type. 

Does that answer the question?

Reply:
Aha! Yes! Thank you.